runtime/frequency/src/lib.rs

```diff
@@ -1110,6 +1118,7 @@ construct_runtime!(
 	{
 		// System support stuff.
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
+		#[cfg(any(not(feature = "frequency-rococo-local"), feature = "all-frequency-features"))]
```

Comment:
Does this mean we cannot run as a parachain locally? Or would we now use `frequency-rococo-testnet` to do that?

Makes me wonder if there is a way to do this based on a cli flag like manual sealing is instead of a feature flag